frontend/src/views/consumer/paymentstatus-modal/index.js

```diff
@@ -91,23 +91,26 @@ export default withTheme(
                       {longText}
                     </Typography>
                     <br />
-                    {query.status == 4 &&
-                      event && (
-                        <Typography
-                          style={{ whiteSpace: 'pre-line' }}
-                          type="largebody"
-                          color={this.props.theme.palette.primaryDeep}
-                        >
-                          {event.contactInformation}
-                        </Typography>
-                      )}
-                  </Content>
-                  {event &&
-                    query.status == 0 && (
-                      <EventCardWrapper bgColor={event.themeColor}>
-                        <EventCard mini event={event} />
-                      </EventCardWrapper>
+                    {query.status == 0 && event && (
+                      <Typography type="largebody">
+                        {event.contactInformation}
+                      </Typography>
+                    )}
+                    {query.status == 4 && event && (
+                      <Typography
+                        style={{ whiteSpace: 'pre-line' }}
+                        type="largebody"
+                        color={this.props.theme.palette.primaryDeep}
+                      >
+                        {event.contactInformation}
+                      </Typography>
                     )}
+                  </Content>
```

Comment:
Please avoid styling changes in the future since this is not a good practice to include as every developer has different style settings in their editor and it will cause changes everytime the other developer creates a pull request for the same file and this can cause difficulties in review also.

Reply:
Yes, will keep this in mind.

Should i revert the style back to the original?

Reply:
No need make sure this from the next pull request.

Reply:
Sure 👍